_maps/RandomRooms/greytopia.dmm

```diff
@@ -0,0 +1,562 @@
+	luminosity = 2
+	},
+/area/greytopia)
+"l" = (
+/obj/item/target,
+/obj/item/gun/energy/laser/chameleon{
+	pixel_y = -8
+	},
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"m" = (
+/obj/structure/bed,
+/obj/item/bedsheet/grey,
+/turf/open/floor/plating{
+	icon_state = "panelscorched"
+	},
+/area/greytopia)
+"n" = (
+/turf/open/floor/plating{
+	icon_state = "platingdmg1"
+	},
+/area/greytopia)
+"o" = (
+/obj/structure/chair,
+/obj/item/device/radio/headset/headset_com{
+	flags_1 = 80;
+	flags_2 = 66;
+	name = "\proper the head of assistants's bowman headset"
+	},
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"p" = (
+/obj/structure/mirror{
+	pixel_x = 5
+	},
+/turf/closed/wall/r_wall,
+/area/greytopia)
+"q" = (
+/obj/structure/sink{
+	dir = 8;
+	pixel_x = -12
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"r" = (
+/obj/effect/decal/cleanable/blood/gibs/torso,
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"s" = (
+/obj/structure/chair{
+	dir = 4
+	},
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"t" = (
+/obj/structure/table,
+/obj/item/reagent_containers/food/snacks/salad/validsalad{
+	pixel_x = -9;
+	pixel_y = 2
+	},
+/obj/item/reagent_containers/food/snacks/salad/validsalad{
+	pixel_x = 7;
+	pixel_y = 1
+	},
+/obj/item/candle/infinite{
+	pixel_y = 9
+	},
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"u" = (
+/obj/item/chair,
+/obj/item/banhammer,
+/obj/effect/decal/cleanable/blood/old,
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"v" = (
+/obj/machinery/computer/security/wooden_tv,
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"w" = (
+/obj/structure/table,
+/obj/item/stack/tile/plasteel{
+	amount = 15;
+	desc = "Extra sharp, they don't make em like this anymore...";
+	throwforce = 15
+	},
+/obj/item/clothing/under/color/grey/glorf{
+	pixel_x = -6;
+	pixel_y = 4
+	},
+/obj/item/candle/infinite{
+	pixel_x = 11;
+	pixel_y = -2
+	},
+/turf/open/floor/plating{
+	icon_state = "platingdmg3"
+	},
+/area/greytopia)
+"x" = (
+/turf/open/floor/plating,
+/area/greytopia)
+"y" = (
+/obj/machinery/computer/arcade/battle{
+	dir = 4;
+	name = "hacked arcade machine";
+	obj_flags = 5
+	},
+/turf/open/floor/plating{
+	luminosity = 2
+	},
+/area/greytopia)
+"z" = (
+/obj/item/chair{
+	dir = 4
+	},
+/obj/item/banhammer,
+/obj/effect/decal/cleanable/blood/old,
+/turf/open/floor/plating,
+/area/greytopia)
+"A" = (
+/obj/structure/chair{
+	dir = 8
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"B" = (
+/turf/open/floor/plating{
+	icon_state = "platingdmg2"
+	},
+/area/greytopia)
+"C" = (
+/obj/structure/chair/comfy{
+	dir = 1
+	},
+/turf/open/floor/plating,
+/area/greytopia)
+"D" = (
+/obj/structure/rack,
+/obj/item/sharpener,
+/obj/item/sharpener,
+/obj/item/shield/riot/roman,
+/obj/item/shield/riot/roman,
+/obj/item/melee/baton/cattleprod{
+	name = "compact stunprod";
+	w_class = 3
```

Comment:
make this a subtype, otherwise the change would revert if you use it to build a beepsky or something.

Reply:
since this isnt even included absent admeme intervention ill probably just cut it and clean it up since apparently every turf has a DV for some reason.